packages/ember-toucan-form/src/components/toucan-form.gts

```diff
@@ -2,6 +2,7 @@ import Component from '@glimmer/component';
 import { hash } from '@ember/helper';
 
 import { HeadlessForm } from 'ember-headless-form';
+import { TrackedAsyncData } from 'ember-async-data';
```

Comment:
This should have be a `type` import. Seems we don't have the linting setup that we have elsewhere to enforce this.